src/server/helpers/render.js

```diff
@@ -16,12 +16,46 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
+// @flow
 
 import * as utils from './utils';
 import Handlebars from 'handlebars';
 
 
-function renderRelationship(relationship) {
+type EntityInRelationship = {
+	bbid: number | string,
```

Comment:
I noticed that you're using integers for the relationship tests but all BBIDs should be strings.

Reply:
Fixed, you're right.